docs/stylesheets/dark-theme.scss

```diff
@@ -1,9 +1,59 @@
-$link_color: #FF7A04;
-$header_background: #FF7A04;
+$accent: #FF7A04;
+
+$link_color: $accent;
+$header_background: $accent;
 $background_color1: #2B2B2B;
 $background_color2: #1E2022;
-$navbar_header_background: $link_color;
+$navbar_header_background: $accent;
+
+/* Extraced fonts from Google */
+/* roboto-regular - cyrillic-ext_greek_cyrillic_vietnamese_latin-ext_greek-ext_latin */
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Roboto'), local('Roboto-Regular'),
+       url('./fonts/roboto-v20-cyrillic-ext_greek_cyrillic_vietnamese_latin-ext_greek-ext_latin-regular.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */
```

Comment:
Could we move the fonts to `docs/fonts` instead of `docs/stylesheets/fonts`? 

Guess that makes more sense...